spec/features/people_skill_search_spec.rb

```diff
@@ -2,34 +2,117 @@
 
 describe :people_skills do
   describe 'People Search', type: :feature, js: true do
-    let(:skill) { skills(:rails).attributes }
+    let(:rails) { skills(:rails).attributes }
 
     before(:each) do
       sign_in auth_users(:user), scope: :auth_user
     end
 
     it 'Should return matching entries to page' do
       visit people_skills_path
-      select skill["title"].to_s, from: 'skill_id'
-      page.find(".star5").click(x: 10, y: 10)
-      page.find("#level").set(3)
+
+      select rails["title"].to_s, from: 'skill_id[]'
+      page.find("#row0_star-label5").click(x: 10, y: 10)
+      page.first(".form-range").set(3)
       expect(page).to have_text('Bob Anderson')
       expect(page).to have_text('Wally Allround')
     end
 
     it 'Should set values of query parameters' do
-      visit(people_skills_path + "?skill_id=#{skill["id"]}&level=3&interest=5")
-      expect(page).to have_select("skill_id", selected: "Rails")
-      expect(page).to have_field("level", with: 3)
-      expect(page).to have_field("interest", with: 5, visible: false)
+      visit(people_skills_path + "?skill_id[]=#{rails["id"]}&level[]=3&interest[0]=5")
```

Comment:
this is quite a manual way of providing url params ...
`people_skills_path` could accept arguments as hash and I would prefer it this way since it makes it more human readable